src/main/java/de/adesso/anki/battle/world/bodies/Rocket.java

```diff
@@ -1,32 +1,22 @@
 package de.adesso.anki.battle.world.bodies;
 
-import java.util.Iterator;
-import java.util.List;
-import java.util.Timer;
-import java.util.TimerTask;
-import java.util.concurrent.TimeUnit;
-
-import org.eclipse.paho.client.mqttv3.MqttException;
-
-import com.states.GameState;
-
 import de.adesso.anki.battle.mqtt.MqttService;
 import de.adesso.anki.battle.util.Position;
 import de.adesso.anki.battle.world.Body;
 import de.adesso.anki.battle.world.DynamicBody;
 import de.adesso.anki.battle.world.World;
+import org.eclipse.paho.client.mqttv3.MqttException;
+
+import java.util.List;
 
 public class Rocket extends DynamicBody {
 //TODO should rockets follow the roadmap ? 
 	// else explode after x seconds for garbage collector
-	private String direction; 
 	
 	// hotfix for driving into own rockets
-	private long timer ; 
-	private boolean active;
+	private long timer;
 	
 	public Rocket (String direction) {

```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused method parameter "direction". ([squid:S1172](https://sonarcloud.io/organizations/adessoag-github/rules#rule_key=squid%3AS1172))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.adesso.anki%3Aanki-battle&pullRequest=22&issues=AWKWIFfR1B6WA4TguGK7&open=AWKWIFfR1B6WA4TguGK7)

[ISSUE_KEY]: AWKWIFfR1B6WA4TguGK7